app/mailers/user_notifications.rb

```diff
@@ -31,6 +33,25 @@ def signup_after_approval(user, opts = {})
                 new_user_tips: tips)
   end
 
+  def suspicious_login(user, opts = {})
+    ipinfo = DiscourseIpInfo.get(opts[:client_ip])
+    location = [ipinfo[:city], ipinfo[:region], ipinfo[:country]].reject { |x| x.blank? }.join(", ")
```

Comment:
Slightly shorted notation

```suggestion
    location = [ipinfo[:city], ipinfo[:region], ipinfo[:country]].reject(&:blank?).join(", ")
```